spec/unit/knife/status_spec.rb

```diff
@@ -44,8 +44,8 @@
       @knife.run
     end
 
-    it "should filter healthy nodes" do
-      @knife.config[:hide_healthy] = true
+    it "should filter by nodes older than some mins" do
+      @knife.config[:hide_by_mins] = 60
```

Comment:
given that hide_by_mins is now variable, it'd be good to make this 59 or 61 everywhere and update the tests accordingly so we're sure no code is still using the old `config[:hide_healthy]` which is fixed to 60.